Rubberduck.UnitTesting/UnitTesting/VBEInteraction.cs

```diff
@@ -67,36 +67,45 @@ public void EnsureProjectReferencesUnitTesting(IVBProject project)
             if (project == null || project.IsWrappingNullReference) { return; }
             var libFolder = IntPtr.Size == 8 ? "win64" : "win32";
             const string libGuid = RubberduckGuid.RubberduckTypeLibGuid;
-            var pathKey = Registry.ClassesRoot.OpenSubKey($@"TypeLib\{{{libGuid}}}\{_rubberduckVersion.Major}.{_rubberduckVersion.Minor}\0\{libFolder}");
-
-            var referencePath = pathKey?.GetValue(string.Empty, string.Empty) as string;
-            string name = null;
-
-            if (!string.IsNullOrWhiteSpace(referencePath))
+            using (var pathKey = Registry.ClassesRoot.OpenSubKey(
```

Comment:
`OpenSubKey` can return null. This needs a null check after `pathKey` is set, and it should be explicitly `Dispose`'d.